storage/range_command.go

```diff
@@ -602,10 +602,11 @@ func (r *Range) InternalPushTxn(batch engine.Engine, ms *proto.MVCCStats, args *
 	}
 
 	if !pusherWins {
+		err := proto.NewTransactionPushError(args.Txn, reply.PusheeTxn)
 		if log.V(1) {
-			log.Infof("failed to push intent %s vs %s using priority=%d", reply.PusheeTxn, args.Txn, priority)
+			log.Info(err.Error())
```

Comment:
it's a valid point. It's going to be handled differently internally since the structured logging code does type assertions. But probably that's what we want more than losing structure by converting here already. I'll change it.